content/Intro to HPC/Module 4 - When should I use a cluster?.md

```diff
@@ -0,0 +1,23 @@
+# Module 4 - When should I use a cluster?
+
+## Long-running processes
+Processes that exceed a typical runtime, often more than 24 hours, can be defined long-running processes, for example:
+- Engineering: tasks such as aerodynamics, structural engineering, and fluid dynamics design and optimization.
+- Financial sector: risk simulations and financial modeling to predict market trends or assess risks.
+- Healthcare: molecular modeling for drug discovery and interaction simulations.
+
+To deal with the long-running processes, checkpointing is usually used at fixed time interval to save progress and prevent data loss, and scheduling tools are needed to organize the job.
+
+## GPU-accelerated workloads
+When considering the use of an HPC cluster for GPU-accelerated workloads, it’s important to evaluate the availability and configuration of GPU resources, like the type and number of GPUs. Then, these GPUs are often set up with MIG (multi-instance GPU) to optimize for varied workloads, allowing multiple users or jobs to utilize the GPU resources efficiently. Furthermore, for jobs requiring multiple full GPUs, special requests should be made to the HPC administrators for reconfiguration.
```

Comment:
Might be worthwhile describing MIG in its own paragraph and going into more detail.
1. What is MIG?
2. How does this help a user (i.e. being able to use 1 large GPU or multiple smaller GPUs)
3. Is MIG dynamic --- can I change the settings for just my job (the answer is "no", the HPC sysadmin sets up the MIG profile and changing it involves rebooting the node)